update.sh

```diff
@@ -17,7 +17,7 @@ echo "Updating certificates..."
 ./certs/update.sh
 
 echo "Fetching and building traefik $VERSION..."
-wget -O traefik https://github.com/emilevauge/traefik/releases/download/$VERSION/traefik
+wget -O traefik https://github.com/containous/traefik/releases/download/$VERSION/traefik
```

Comment:
wget has a lot of implicit behavior for better ergonomics on the 80% use case. I know for HTTP errors it will output error codes, but I'm not sure about TCP errors. I think generally it performs retries, but again, I'm more familiar with curl.